fdbclient/S3Client.actor.cpp

```diff
@@ -283,7 +291,7 @@ ACTOR static Future<Void> copyUpFile(Reference<S3BlobStoreEndpoint> endpoint,
 		tags[S3_CHECKSUM_TAG_NAME] = checksum;
 		wait(endpoint->putObjectTags(bucket, objectName, tags));
 
-		TraceEvent(SevInfo, "S3ClientCopyUpFileEnd")
+		TraceEvent(s3PerfEventSev(), "S3ClientCopyUpFileEnd")
```

Comment:
Should this match the serverity for S3ClientCopyUpFileStart or are you thinking that we might want to log the end of upload but not the start?

Reply:
I think you removed the PerfEventSev which I think is a good idea. One is enough.